tests/lib/rules/id-blacklist.js

```diff
@@ -65,6 +65,11 @@ ruleTester.run("id-blacklist", rule, {
             code: "var obj = { key: foo.bar };",
             options: ["f", "fo", "fooo", "b", "ba", "barr", "bazz", "bingg"]
         },
+        {
+            code: "const {foo: bar} = baz",
```

Comment:
Can we also add tests for nested destructuring assignments? Example:

```js
const { foo: { bar: baz } } = baz;
```

Reply:
Would also be good to add tests for destructured function parameters (including nested `ObjectPattern`s):

```js
function foo({ bar: baz }) {}
```